tests/test_stoplight_elements_docs.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+app = FastAPI(title="White Shuli 2")
+
+
+@app.get("/pita/shuli")
```

Comment:
Recommend to use very neutral language in tests, like 
```
@app.get("/a/b")
def get_a_and_b():
    return {"a": "b"}
```
Otherwise this might confuse future developers who have no idea what pita shuli is.